### PR TITLE
intial no exercises notice

### PR DESCRIPTION
**Pull request by pandafulmanda**
screen shot 2015-11-24 at 5 37 07 pm

handles both
- [x] task.steps is empty for whatever reason
- [x] module requested is actually an intro module and will error as in openstax/tutor-server#836
- [x] clean up a little

### CODE REVIEW

**Comment by nathanstitt** [comment]
:+1: Awesomely appropriate font awesome icon :grin:

File: resources/styles/components/task/index.less [hunk 1]
```diff
@@ -14,14 +14,18 @@
...
 
-  .chapter-section-prefix {
+  .chapter-section-prefix, .no-title {
     display: inline-block;
     margin-right: 1rem;
   }
 
-  a {
-    color: @openstax-neutral-light;
+  &:not(.no-title) {
```
Comment (philschatz) [inline resources/styles/components/task/index.less]: could this be `.has-title`? That way we don't not need no double negatives? :smile:

File: src/task/no-exercises.cjsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+React = require 'react'
+
+NoExercises = React.createClass
+  displayName: 'NoExercises'
+  render: ->
+    <div>
```
Comment (philschatz) [inline src/task/no-exercises.cjsx]: can you add a class to the element for possible selenium testing?

**Comment by pandafulmanda** [comment]
@philschatz thanks for the comments, i've addressed them

**Pull request merged**